config/containers/logging/local.md

```diff
@@ -50,6 +57,15 @@ The `local` logging driver supports the following logging options:
 
 | Option      | Description                                                                                                                                                                                                   | Example  value                           |
 |:------------|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:-----------------------------------------|
-| `max-size`  | The maximum size of each binary log file before rotation. A positive integer plus a modifier representing the unit of measure (`k`, `m`, or `g`). Defaults to `20m`.                                          | `--log-opt max-size=10m`                 |
-| `max-file`  | The maximum number of binary log files. If rotating the logs creates an excess file, the oldest file is removed. **Only effective when `max-size` is also set.** A positive integer. Defaults to `5`.         | `--log-opt max-file=5`                   |
-| `compress`  | Whether or not the binary files should be compressed. Defaults to `true`                                                                                                                                      | `--log-opt compress=true`              |
+| `max-size`  | The maximum size of the log before it is rolled. A positive integer plus a modifier representing the unit of measure (`k`, `m`, or `g`). Defaults to 20MB.                                          | `--log-opt max-size=10m`                 |
```

Comment:
"Defaults to 20MB" ---> "Defaults to 20m"

Reply:
Got it - thank you!